### PR TITLE
Replace hyphen with underscore in ahSettingsFile (#86)

### PR DESCRIPTION
**Pull request by ssu-carlsoni**
Fixes # 
--------
#86 

Motivation
----------
Acquia Cloud does not support hyphens in db names.  FilePaths::ahSettingsFile handles periods but not hyphens.

Proposed changes
---------
Ensure FilePaths::ahSettingsFile return the correct settings file when a site_name contains a hyphen

Alternatives considered
---------
A hyphen is valid character in domain names.

Testing steps
---------
Add a multisite with a hyphen in the name to a project on Acquia Cloud using Acquia Recommended Settings.
example: childrens-school.domain.com

### CODE REVIEW

File: src/FilePaths.php [hunk 1]
```diff
@@ -28,6 +28,8 @@ public static function ahSettingsFile(string $ah_group, string $site_name) {
 
     // Acquia Cloud does not support periods in db names.
     $site_name = str_replace('.', '_', $site_name);
+    // Acquia Cloud does not support hyphens in db names.
+    $site_name = str_replace('-', '_', $site_name);
```
Comment (danepowell) [inline src/FilePaths.php:32]:
```suggestion
    // Acquia Cloud does not support periods or hyphens in db names.
    $site_name = str_replace(['.','-'], '_', $site_name);
```

**Review by danepowell — CHANGES_REQUESTED** [review]
I think this can be simplified. Check my suggestion, make sure tests pass, then I'm happy to merge.

**Comment by ssu-carlsoni** [comment]
Thank you @danepowell. I learned str_replace can accept arrays for search and/or replace.

**Comment by danepowell** [comment]
Thanks. I think there's a CI outage right now but I'll circle back to this tomorrow.

**Pull request merged**